common/decklist.cpp

```diff
@@ -500,32 +547,36 @@ bool DeckList::loadFromStream_Plain(QTextStream &in)
             isSideboard = true;
         }
 
+        if (line.trimmed().isEmpty())
+            continue; // The line was "    " instead of "\n"
```

Comment:
Can you explain this a little bit more? Is it a check for empty SB: lines like
```
SB:       
```
Isn't line supposed to have already be trimmed by `simplified()` above?